src/site/antora/modules/ROOT/pages/manual/appenders/message-queue.adoc

```diff
@@ -423,19 +423,21 @@ This example cannot be configured using Java properties.
 [#JmsAppender]
 == JMS Appender
 
-The JMS Appender sends the formatted log event to a
-https://jakarta.ee/specifications/messaging/2.0/[Jakarta Messaging API]
+The JMS Appender sends a formatted log event to a
+https://jakarta.ee/specifications/messaging/3.0/[Jakarta] or https://jakarta.ee/specifications/messaging/2.0/[Javax] Messaging API
 destination.
 
 [IMPORTANT]
 ====
 As of Log4j `2.17.0` you need to enable the JMS Appender **explicitly** by setting the
 xref:manual/systemproperties.adoc#log4j2.enableJndiJms[`log4j2.enableJndiJms`]
 configuration property to `true`.
-
-Due to breaking changes in the underlying API, the JMS Appender cannot be used with Jakarta Messaging API 3.0 or later.
 ====
 
+For Jakarta, use the `JMS-Jakarta` element name in the `log4j-jakarta-jms` Maven module.
+
+For Javax, use the `JMS-Javax` element name; the names `JMS`, `JMSQueue`, and `JMSTopic` are provided for backward compatibility.
+
```

Comment:
The collapsible UI to too lame to use IMO. It's hard to tell how much is expanded, maybe if it expanded into something with a border it would help. I did not want to click on the "Click here" text because I did not want to leave the page. 

The true uselessness of the collapsible thingy is that when you search for text on a page in a web browser (Firefox in my case), the browser does not find the text inside the collapsed text, so using the collapsible thingy an anti-pattern as implemented (IMO). Or maybe Firefox found it but you certainly don't see it.

The new site is hard enough to use without this kind of UI to make it worse 😞... IMO